routers/tournamentStatusRouter.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+const express = require('express')
+
+const mhubConnection = require('../Utils/mhubConnection')
+const getMatchLogic = require('../logic/getMatchLogic')
+
+const { authroizationMiddlware } = require('@first-lego-league/ms-auth')
+
+const adminAction = authroizationMiddlware(['admin', 'development'])
+
+exports.getRouter = function () {
+  const router = express.Router()
+
+  router.get('/current', (req, res) => {
+    getMatchLogic.getCurrentMatch().then(data => {
+      if (data) {
+        mhubConnection.publishUpdateMsg('CurrentMatches', data)
+        res.send(data)
+      } else {
+        res.sendStatus(404)
+      }
+    })
+  })
+
+  router.get('/upcoming', (req, res) => {
+    Promise.all(getMatchLogic.getNextMatches()).then(data => data.filter(data => data)).then(data => {
+      if (data && data.length > 0) {
+        mhubConnection.publishUpdateMsg('UpcomingMatches', data)
+        res.send(data)
+      } else {
+        res.sendStatus(404)
+      }
+    })
+  })
+
+  router.get('/matchNumber', (req, res) => {
+    res.json(getMatchLogic.getMatch())
```

Comment:
What this route is doing? get match needs parameter.
Either way, there is a method like that in the crudRouter

Reply:
So what is the difference bteween /current and /matchNumber?